pulse2percept/models/granley2021.py

```diff
@@ -0,0 +1,559 @@
+"""`BiphasicAxonMapModel`"""
+from attr import Attribute
+import numpy as np
+from copy import deepcopy
+import sys
+
+from numpy.core.fromnumeric import size
```

Comment:
Some of these imports seem to be unused, e.g. `Attribute`, `deepcopy`, `size`,  `load_nanduri2012`. Would be good to remove them to avoid unnecessary dependencies